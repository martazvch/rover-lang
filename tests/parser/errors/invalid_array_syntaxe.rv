// Variable declaration are used to synchronize the parser

array[5
var a = 0
[1 56
var a = 0
[1, 3

