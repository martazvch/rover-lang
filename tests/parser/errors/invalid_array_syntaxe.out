missing_array_close_bracket
